package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "github-ci-bot",
+  "name": "kie-ci-bot",
```

Comment:
@Kaitou786 I'm missing github somehow, it looks like a very generic bot here. What about `kie-github-ci-bot`?

Reply:
Let me see, if we can take this.

Reply:
other option would be `kie-github-bot`

Reply:
Both are available, wdyt `kie-github-bot` or `kie-github-ci-bot` ?

Reply:
I think `kie-github-ci-bot` is more precise, ok?

Reply:
Ok, updating.